src/main/java/edu/harvard/dbmi/avillach/dictionary/dashboarddrawer/DashboardDrawerService.java

```diff
@@ -15,7 +15,7 @@ public class DashboardDrawerService {
     private final String dashboardLayout;
 
     @Autowired
-    public DashboardDrawerService(DashboardDrawerRepository repository, @Value("${dashboard.layout.type}") String dashboardLayout) {
+    public DashboardDrawerService(DashboardDrawerRepository repository, @Value("${dashboard.layout.type:default}") String dashboardLayout) {
```

Comment:
Since there is no other implementation or requirements for other projects will just make this layout default.  I do not like setting layout type to bdc anyways.